distributed/core.py

```diff
@@ -64,21 +62,6 @@
     Coro = Coroutine[Any, Any, T]
 
 
-cache_loads: LRU[bytes, Callable[..., Any]] = LRU(maxsize=100)
-
-
-def loads_function(bytes_object):
```

Comment:
cc @madsbk - It looks like we were using this function in dask-cuda (https://github.com/rapidsai/dask-cuda/issues/1219)

Reply:
Yes, we use it for its caching feature but I don't think it is needed.